subhub/customer.py

```diff
@@ -80,16 +80,21 @@ def existing_or_new_customer(
         return create_customer(
             subhub_accouunt, user_id, email, source_token, origin_system, display_name
         )
+    logger.info("existing customer", account=db_account)
```

Comment:
this is a theme you will see:
why not `db_account=db_account`?

Reply:
valid, fixed.